src/cordova/Api.js

```diff
@@ -63,29 +63,25 @@ Api.createPlatform = function (destination, config, options, events) {
     events.emit('log', '\tPath: ' + destination);
     events.emit('log', '\tName: ' + projectName);
 
-    shell.mkdir(destination);
+    shell.mkdir('-p', destination);
 
     // move a copy of our api to the new project
     shell.cp('-r',apiSrcPath, destination);
 
-    // move our node_modules
```

Comment:
unrelated deletion?

Reply:
No, that's intended. Part of 22b9d773bc80003d9084772838cdd5de66c94ed8

Reply:
... which is best obsolete 🥇